Northstar.Coop/mod/scripts/vscripts/sh_coop_custom_precache.gnut

```diff
@@ -0,0 +1,20 @@
+// copied from sh_northstar_custom_precache.gnut
+untyped
+global function CoopCustomPrecache
+
+void function CoopCustomPrecache()
+{
+	// PrecacheWeapon( "mp_weapon_peacekraber" )
```

Comment:
I don't see an issue with loading the peacekraber
```suggestion
	PrecacheWeapon( "mp_weapon_peacekraber" )
```

Reply:
the peacekraber pr hasn't been merged yet :(
#398